### DISCUSSION TITLE
Issue: Does not alert player when they reach aquired limit.

### DISCUSSION THREAD

**Discussion by TomLewis** [GriefPrevention/GriefPrevention#108] [Ideas]
Somehow 2 of my players managed to reach my very high acquired limit for claim blocks, it does not tell them when they hit the limit, it just stops them getting more. Could we please have this added.

**Comment by RoboMWM** [comment]
Things to consider in addon - when/where to notify player (claimlist? On accrual event? on claim create/modify? on holding shovel? etc.), and what threshold.

**Comment by TomLewis** [comment]
Thats a good point! when would you do this, I would suggest when they log in, they get their /mail this way too, so they can see its happeend and they will keep getting that when they log in.
But it cant just be for that user, it needs to be logged/alerted to staff too somehow, so you know if there is an issue or cheating claims or just in general. So maybe it could be the same for staff too, if there are people that have reached the max limit maybe have a permission to show that when staff log in too?

`X people have reached the Griefprevention acquired limit`

Then you can click that message and it just lists them in chat/gui with heads.

Another option is when claims are sent every 10 mins ping the message to staff only? otherwise it could get spammy. I do like the idea of it showing on claimslist too, as this is an overview of everything claims.

**Comment by TomLewis** [comment]
Did this get implemented anywhere?
